README.md

```diff
@@ -21,10 +21,13 @@ For now, Wasm and Wasi don't have a support for asynchronous and multithreading
 
 ### Network components
 
-As mentioned before, Wasm can't handle Async calls, making it challenging to deal with networks. The current state of wasi development doesn’t include native support for Sockets and Http requests, but the community have created some workarounds that enabled some of these features. These are experimental and not recommended to be used in a production environment but serve as a preview and experience on what using the network on WebAssembly might look like before it is implemented and integrated into the main project.
+As mentioned before, Wasm can't handle Async calls, making it challenging to deal with networks. The current state of Wasi development doesn’t include native support for Sockets and Http requests, but the community have created some workarounds that enabled some of these features. These are experimental and not recommended to be used in a production environment but serve as a preview and experience on what using the network on WebAssembly might look like before it is implemented and integrated into the main project.
 
 ## Wasi Debug Echo
```

Comment:
More background on what an Akri Discovery Handler is needed. Also title this section Wasm  -- not Wasi right? -- Debug Echo Discovery Handler.

Reply:
Thanks, will develop more on the Discovery Handler objective. I was planning to call it Wasi, but do you think Wasm fits better?